src/main/scala/viper/carbon/Carbon.scala

```diff
@@ -94,5 +94,11 @@ class CarbonConfig(args: Seq[String]) extends SilFrontendConfig(args, "Carbon")
     noshort = true
   )
 
+  val desugarPolymorphicMaps = opt[Boolean]("desugarPolymorphicMaps",
+    descr = "Do not use polymorphic maps in the Boogie encoding and instead desugar them (default: true).",
+    default = Some(true),
```

Comment:
TODO: change default to false before merging (s.t. with the default options the setting is the same as before)